### PR TITLE
Fix ruff exclude

### PR DESCRIPTION
**Pull request by snaselj**
# Closes: NaN

## What's Changed

- Fixed ruff excludes to use per-file excludes instead of global excludes.
- Align ruff with previous pydocstyle checks.

### CODE REVIEW

File: nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml
Comment (itdependsnetworks) [inline nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml:150]: Does just "D" work?

Comment (snaselj) [reply nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml:150]: I'm not sure with this. There is another set of rules prefixed with "DTZ" (flake8-datetimez), and I can't find, whether just "D" includes "DTZ" or not, in the official documentation. I would rather keep it as is.

Comment (itdependsnetworks) [reply nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml:150]: As we move over to ruff more and more, my preference would be to find out for sure. I won't hold up this PR, just letting my preference be known.

Comment (itdependsnetworks) [reply nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml:150]: Per [ruff docs](https://docs.astral.sh/ruff/configuration/), this seems like it is fine

```toml
[lint]
# Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
# Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
# McCabe complexity (`C901`) by default.
select = ["E4", "E7", "E9", "F"]
ignore = []
```

It states to enable Pyflakes, not to enable ones that start with F.

Comment (snaselj) [reply nautobot-app-chatops/{{ cookiecutter.project_slug }}/pyproject.toml]: Replaced with "D" only.

**Comment by snaselj** [comment]
Can I get approval pls., if everything is fine? @whitej6 @itdependsnetworks

**Review by itdependsnetworks — APPROVED** [review]

**Pull request merged**
